Dockerfile

```diff
@@ -1,6 +1,6 @@
 #/********************************************************************************
-#* Copyright (c) 2022 T-Systems International GmbH
-#* Copyright (c) 2022 Contributors to the CatenaX (ng) GitHub Organisation
+#* Copyright (c) 2022, 2023 T-Systems International GmbH
+#* Copyright (c) 2022, 2023 Contributors to the CatenaX (ng) GitHub Organisation
```

Comment:
I think this is still wrong? The catenax ng github org doesn'th ave contrubion copyright

Reply:
Added the correct header